plugin/db/clickhouse/clickhouse.go

```diff
@@ -703,22 +703,22 @@ const (
 )
 
 // Dump dumps the database.
-func (driver *Driver) Dump(ctx context.Context, database string, out io.Writer, schemaOnly bool) error {
+func (driver *Driver) Dump(ctx context.Context, database string, out io.Writer, schemaOnly bool) (string, error) {
 	txn, err := driver.db.BeginTx(ctx, &sql.TxOptions{})
 	if err != nil {
-		return err
+		return "{}", err
```

Comment:
Handle it in the store instead of everywhere else.

https://github.com/bytebase/bytebase/blob/main/store/sheet.go#L282